src/main/resources/camunda/claimant_response_spec.bpmn

```diff
@@ -63,9 +63,10 @@
       <bpmn:outgoing>Flow_ADMIT_PART_SETTLE_CLAIM</bpmn:outgoing>
       <bpmn:outgoing>Flow_Flow_PAY_IMMEDIATELY</bpmn:outgoing>
       <bpmn:outgoing>Flow_0f1j5lb</bpmn:outgoing>
+      <bpmn:outgoing>Flow_0fq9s3w</bpmn:outgoing>
     </bpmn:exclusiveGateway>
     <bpmn:sequenceFlow id="Flow_FULL_DEFENCE_NOT_PROCEED" name="Applicant confirms not to proceed(full/part admit)" sourceRef="Gateway_PROCEED_OR_NOT_PROCEED" targetRef="ProceedOfflineForResponseToDefence">
-      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">${flowState == "MAIN.FULL_ADMIT_NOT_PROCEED" || flowState == "MAIN.PART_ADMIT_NOT_PROCEED" || flowState == "MAIN.PART_ADMIT_PAY_IMMEDIATELY"}</bpmn:conditionExpression>
+      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">${flowState == "MAIN.FULL_ADMIT_NOT_PROCEED" || flowState == "MAIN.PART_ADMIT_NOT_PROCEED" || (flowState == "MAIN.PART_ADMIT_PAY_IMMEDIATELY" &amp;&amp; (empty flowFlags.JO_ONLINE_LIVE_ENABLED || !flowFlags.JO_ONLINE_LIVE_ENABLED))}</bpmn:conditionExpression>
```

Comment:
![Screenshot 2025-01-09 at 11 02 02](https://github.com/user-attachments/assets/c4ec316c-f1e4-47e9-b122-b21e00ffec6d)

could we add a label to update that we've added a flag check?
something like the above?

Reply:
feel free to rename it however, but it'll be easier to see that this path would not be used if jo live is enabled.